docs/architecture.md

```diff
@@ -36,7 +34,6 @@ JKAN is a [Jekyll](http://jekyllrb.com/) site that uses JavaScript for enhanced
 | `_includes` | HTML partials that are included in layouts. `display` is used by [schema files](/docs/configuration.md) |
 | `_layouts` | Page templates |
 | `_organizations` | A directory of `.md` files containing organization information in YAML front-matter |
```

Comment:
Should we still include this line but with the new file? Ditto for the directory structure above